Header header
# CarSim's Vx_L2 (km/h)
float32 rear_left_encoder
# CarSim's Vx_R2 (km/h)
float32 rear_right_encoder
